.github/workflows/test-check.yaml

```diff
@@ -23,6 +23,6 @@ jobs:
             virtualenv venv
             source venv/bin/activate
         - name: "⚙️ Install dependencies"
-          run: pip3 install .[dev]
+          run: pip3 install .[dev,big_model]
```

Comment:
can the name of the optional install be `accelerate`?

I think the package is well known enough